languages/python/python_test.go

```diff
@@ -26,56 +31,58 @@ func TestPythonImports(t *testing.T) {
 			Name: "relative",
 			Src:  `from a.b import x`,
 			Exp: []string{
-				"./a/b",
+				"a.b",
 			},
 		},
 		{
 			Name: "relative dot",
 			Src:  `from .a.b import x`,
 			Exp: []string{
-				"./a/b",
+				".a.b",
 			},
 		},
 		{
 			Name: "relative dot 2",
 			Src:  `from ..a.b import x`,
 			Exp: []string{
-				"../a/b",
+				"..a.b",
 			},
 		},
 		{
 			Name: "relative dot 3",
 			Src:  `from ...a.b import x`,
 			Exp: []string{
-				"../../a/b",
+				"...a.b",
 			},
 		},
 		{
 			Name: "relative dots",
 			Src:  `from ... import x`,
 			Exp: []string{
-				"../../",
-			},
-		},
-		{
-			Name: "relative named",
-			Src:  `from a.b import x as b`,
-			Exp: []string{
-				"./a/b",
+				"...",
 			},
 		},
+		// We have to think how to fix it (maybe posprocess and get rid of aliases).
```

Comment:
I think it's because the query is not exactly correct and matches the whole subtree in case of aliases.
Those imports are common, so we need to post-process it or change the query.

Reply:
I think the query `((import_from_statement module_name: (dotted_name (identifier)) @name))` is correct, but because we have a next query 
`((aliased_import name: (dotted_name (identifier)) @name))` the alias goes there as well.
Or maybe we can add another query which can eliminate the alias, but so far all my tries didn't succeeded

Reply:
we can also leave a generic query only, and then query again for that particular case

Reply:
Yep, I thought about it.

Reply:
I already have a fix - we just don't need to iterate through all Matches. Just take the first one and everything will be fine.
I double check different tests  and push it.

Reply:
Unfortunately, the fix didn't work for all cases.
The problem for the case:
`from a.b import x as b`
is, it produces following node:
```
(module (import_from_statement module_name: (dotted_name (identifier) (identifier)) name: (aliased_import name: (dotted_name (identifier)) alias: (identifier))))
```

so, it satisfies `import_from_statement`and in next iteration (as a next match) it satisfies `aliased_import` (so it returns x).

we cannot get rid of `aliased_import` query, because we need to handle cases like:
`import a.b as c`

Anyway, I'll try to fix it in next PR. Maybe we can just keep basic queries and post-process them so if there is import inside import we will ignore it.

Reply:
Yeah, I did something for [java](https://github.com/src-d/imports/pull/2/files#diff-5cc8d5c7ae108ad2c1e83a64ee177ca1R55). Let's just make it work and then figure out how we can improve it.